code/game/atoms_movable.dm

```diff
@@ -848,6 +848,9 @@
 /atom/movable/proc/do_attack_animation(atom/target, atom/tool)
 	set waitfor = 0
 
+	if(!tool)
```

Comment:
I'm not sure if this is good though, we never would have found the other examples had they not runtimed (and this would have prevented them from runtiming)